dom.bs

```diff
@@ -4533,33 +4533,82 @@ each <a for=tree>descendant</a> <a>exclusive <code>Text</code> node</a> <var>nod
 
 <div class=impl>
 
+<div algorithm>
 <a lt="Other applicable specifications">Specifications</a> may define
 <dfn export id=concept-node-clone-ext>cloning steps</dfn> for all or some <a for=/>nodes</a>. The
-algorithm is passed <var>copy</var>, <var>node</var>, <var>document</var>, and an optional
-<i>clone children flag</i>, as indicated in the <a lt="clone a node">clone</a> algorithm.
+algorithm is passed <var ignore>node</var>, <var ignore>copy</var>, and <var ignore>subtree</var> as
+indicated in the <a>clone a node</a> algorithm.
 
-<p class=note>HTML defines <a>cloning steps</a> for <{script}> and <{input}>
-elements. SVG ought to do the same for its <{script}> elements, but does not call this out
-at the moment.
+<p class=note>HTML defines <a>cloning steps</a> for several elements, such as <{input}>, <{script}>,
+and <{template}>. SVG ought to do the same for its <{script}> elements, but does not.
+</div>
 
-<p>To <dfn export id=concept-node-clone lt="clone a node" local-lt="clone">clone</dfn> a
-<var>node</var>, with an optional <var>document</var> and <i>clone children flag</i>, run these
-steps:
-<!-- This algorithm is used by dom-Node-cloneNode, dom-Document-importNode,
-dom-Range-extractContents, dom-Range-cloneContents -->
+<div algorithm="clone a node">
+<p>To <dfn export id=concept-node-clone>clone a node</dfn> given a <a for=/>node</a> <var>node</var>
+and an optional <a for=/>document</a> <dfn export for="clone a node"><var>document</var></dfn>
+(default <var>node</var>'s <a for=Node>node document</a>), boolean
+<dfn export for="clone a node"><var>subtree</var></dfn> (default false), and
+<a for=/>node</a>-or-null <dfn export for="clone a node"><var>parent</var></dfn> (default null):
 
 <ol>
- <li><p>If <var>document</var> is not given, then set <var>document</var> to <var>node</var>'s
- <a for=Node>node document</a>.
+ <li><p><a for=/>Assert</a>: <var>node</var> is not a <a for=/>document</a> or <var>node</var> is
+ <var>document</var>.
 
- <li><a for=/>Assert</a>: <var>node</var> is not a <a for=/>document</a> or <var>node</var> is
+ <li><p>Let <var>copy</var> be the result of <a>cloning a single node</a> given <var>node</var> and
  <var>document</var>.
 
+ <li><p>If <var>parent</var> is non-null, then <a for=/>append</a> <var>copy</var> to
+ <var>parent</var>.
+
+ <li><p>Run any <a>cloning steps</a> defined for <var>node</var> in
+ <a>other applicable specifications</a> and pass <var>node</var>, <var>copy</var>, and
+ <var>subtree</var> as parameters.
```

Comment:
Gecko runs cloning steps as part of cloning a single node, and I think webkit does that too https://searchfox.org/wubkat/rev/72adbb9f99534551a728acb270d4f111da5e9515/Source/WebCore/dom/Element.cpp#648,655,663 
and chrome too https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/dom/element.cc;l=755;drc=f5e280b6c11f41545154266cd2317d3c730e9b30;bpv=0;bpt=1